Analysis/analysis.tex

```diff
@@ -259,6 +269,7 @@ \section{Minor SM Backgrounds}
 These processes, collectively denoted as minor SM backgrounds, contribute in the signal region when the jet energy is severely mismeasured or the leptons fail to be reconstructed resulting in large \met in the signal region. 
 However, the \met is typically aligned with the photon or one of the jets in such cases, and therefore various selections on the kinematic relations between the \met, photons, and jets are used to reduce these backgrounds to a manageable rate. 
 The estimates for all five processes are taken from \MGvATNLO simulations at LO in QCD and are listed in Tables~\ref{tab:yield_mask_horizontal} and~\ref{tab:yield_mask_vertical}.
+\PH{What did you do for the MET cut uncertainty in the g+Jets. I honestly think you should at least split this MC out and quote a large unc. on it. }
```

Comment:
this is probably the more correct thing to do, but even when we had all the MC split out by type it didn't majorly change the results, so going to leave it like this for the thesis.